components/mosart/bld/namelist_files/namelist_defaults_mosart.xml

```diff
@@ -32,4 +39,34 @@ for the CLM data in the CESM distribution
 <frivinp_rtm rof_grid="r05" >rof/mosart/MOSART_global_half_20180721a.nc</frivinp_rtm>
 <frivinp_rtm rof_grid="NLDAS" >rof/mosart/MOSART_NLDAS_8th.nc</frivinp_rtm>
 
+<!-- WRM -->
+
+<paraFile rof_grid="NLDAS" >rof/mosart/US_reservoir_8th_NLDAS3_c20161220_updated_20170314.nc</paraFile>
```

Comment:
this file needs to be uploaded to E3SM input data server

Reply:
Added to Cori:/global/project/projectdirs/acme/inputdata/rof/mosart
Together with this file (for NLDAS), I also added one fore global half degree.